docs/manual/java/gettingstarted/WhatsNext.md

```diff
@@ -0,0 +1,22 @@
+# What's Next?
+
+After working through "Getting started with Lagom", follow the path that make the most sense to you:
+
+Learn how to integrate with Eclipse:
+
+* [[Import an existing Maven project into Eclipse|EclipseMavenInt]] 
+* [[Create a new Eclipse project using the Lagom Maven archetype|EclipseMavenNew]]
+* [[Import an existing sbt build into Eclipse|EclipseSbt]]
+    
+Learn how to integrate with Intellij:
+
+* [[Import an existing Maven project into Intellij|intellijMaven]]
+* [[Import an existing sbt build into Intellij|IntellijSbtJava]]
+    
+Learn more about Lagom:
+
+* [[Core concepts|CoreConcepts]]
+* [[Using the service locator|ServiceLocator]]
+* [[Using the message broker|MessageBrokerAPI]]
```

Comment:
I think this should link to `MessageBroker`. Currently, the link is broken.

There is a page called `MessageBrokerApi` and this link with `MessageBrokerAPI` might work on Windows but doesn't on Mac/Linux and probably won't in production. In any case, I think `MessageBroker` is more introductory, so a better next step from here.